modules/dataquery/jsx/nextsteps.tsx

```diff
@@ -31,102 +49,121 @@ function NextSteps(props: {
         ? 'Modify Filters'
         : 'Add Filters';
     switch (props.page) {
-    case 'Info':
+    case Tabs.Info:
         if (canRun) {
             // A previous query was loaded, it can be either
             // modified or run
             steps.push(<ButtonElement
                     label={fieldLabel}
                     columnSize='col-sm-12'
                     key='fields'
-                    onUserInput={() => props.changePage('DefineFields')}
+                    onUserInput={() => props.changePage(Tabs.Fields)}
             />);
```

Comment:
This button element seems to be repeated a lot, can it be factorized ?

Reply:
I didn't change the button, just the argument to the callback which had the wrong type